src/components/ChartTooltip.jsx

```diff
@@ -48,7 +48,7 @@ const ChartTooltip = ({ active, payload, label, percentage, precision = {}, date
           <thead className='sr-only'>
             <tr>
               <th>Color</th>
-              <th>Lineage</th>
+              <th>Name/Lineage</th>
```

Comment:
I hadn't even taken in that we had this accessibility - super cool! Lineage alone would be accurate, even for variants, but it doesn't matter either way.

Reply:
while it's technically correct to add the header, I don't think the tooltip itself is accessible :(

Reply:
Definitely better than nothing!